packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -240,8 +240,20 @@ exports.submit = async (req, res) => {
 	const journey = getJourney(journeyResponse);
 	if (journey.isComplete()) {
 		await submitQuestionnaireResponse(journeyResponse.journeyId, encodeURIComponent(referenceId));
-		return res.render('./dynamic-components/submission-screen/index');
+		return res.redirect(
+			'/manage-appeals/' + encodeURIComponent(referenceId) + '/questionnaire-submitted/'
+		);
 	}
 	// return error message
 	return res.sendStatus(400);
 };
+
+exports.submitted = async (req, res) => {
+	const journeyResponse = res.locals.journeyResponse;
+	const journey = getJourney(journeyResponse);
+	if (journey.isComplete()) {
+		return res.render('./dynamic-components/submission-screen/index');
+	}
+	// return error message and redirect
+	return res.status(400).render('./error/not-found.njk');
```

Comment:
Might be nice to swap these so the happy path isn't nested

```suggestion
	if (!journey.isComplete()) {
		// return error message and redirect
	  return res.status(400).render('./error/not-found.njk');
	}
  return res.render('./dynamic-components/submission-screen/index');
```

you could even add an `isNotComplete` method to the journey class for added readability